doc/source/news/15.md

```diff
@@ -1,5 +1,23 @@
 # News - 15 series
 
+<!-- prettier-ignore-start -->
+(release-15-0-2)=
+## Release 15.0.2 - 2025-02-21
+<!-- prettier-ignore-end -->
+
+### Fixes
+
+#### {doc}`/reference/columns/index`: Fixed an {ref}`offline-index-construction` bug
+
+Groonga may ingnore normalizer/tokenizer/tokenfilter options of target index when offline index construction execute parallel
+
+If happend when:
+
+- Execute offline index construction with parallel
+- Use Normalizer/Tokenizer/TokenFilter options
+
+However, if we use NormalizerTable, offline index construction occurs error by this issue.
```

Comment:
```suggestion
If {ref}:`${NormalizerTable link}` is used and this happens, the offline index construction is failed. Because {ref}:`${NormalizerTable link}` has a required parameter. If options are ignored, the required parameter is missing.
```